accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+        "Mono.Android.LLVM.Aot.armv7a": {
+            "version" : "8.4.7",
+            "kind": "sdk",
+            "alias-to": {
+                "osx-x64": "Mono.Android.LLVM.Aot.armv7a.MacHost",
+                "win-x64": "Mono.Android.LLVM.Aot.armv7a.Win64Host"
+            }
+        }
+    }
+}
+```
+
+# Side-by-Side Platform Updates
+
+In .NET 5, the [TFM is used to specify the .NET version](https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md) (e.g. .NET 5, .NET 6), and may also be used to specify a target platform (e.g. iOS, Android, WinUI) to access platform-specific API. If the TFM specifies a platform, it also implicitly or explicitly specifies a *Platform API Version*. Developers are generally expected to use the latest platform API version, specify a minimum OS version, and [use runtime checks to guard use of APIs from newer OS versions](minimum-os-version/minimum-os-version.md). Templates burn in the most recent OS API version so that new projects start out in the recommended configuration, warning-free and with access to the latest APIs.
+
+It is strongly recommended that updates *within* an SDK feature band only contain low-risk servicing updates, not new features. This means that developers can feel safe installing servicing updates. They know that it's not going to allow them to accidentally make the project depend on APIs that will break it for other members on their team who are not as up to date.
+
+However, there is a major exception to this: OS releases do not coincide with .NET releases, so we need to be able to release platform API updates between .NET releases.
+
+The solution is to ship two versions of the platform APIs, runtime, build logic, and tools, and have the build logic determine which to use [based on the platform API version](workload-resolvers.md#side-by-side-sdk-support-for-new-os-releases). This essentially makes these platform API updates opt-in on the project level: projects can express their dependency on the updated SDK or workload, and build system can then provide better errors when the SDK has not been updated appropriately.
+
+> ***NOTE:*** Projects depending on any new SDK functionality (e.g. those that might appear in 3.1.200 series compared to 3.1.100 series) would have similar issues, and those dependencies can be handled (in a limited fashion) with global.json. However, platform updates are expected to be far more impactful than other changes that might be shipped in SDK updates, may happen between SDK series releases, and can be expressed in a more clear and comprehensible way using the OS platform versions, so we address them specifically.
+
+## Side-by-Side Manifest Pattern
+
+The pattern is to duplicate the build workloads and the packs they use, and add versioned suffixes to one of the copies of each, for example `ios-14` and `ios`. Note that duplicate packs may be created using aliases to synthesize packs that don't correspond to real NuGet IDs.
```

Comment:
I thought our planned policy was to introduce 15.0 as opt-in until we moved to a band, at which point 15.0 would be the only option. This seems like 15.0 being the default right away.

Reply:
Fleshed out the example a bunch